src/main/java/com/learning/libraryManagement/controller/BooksController.java

```diff
@@ -1,4 +1,28 @@
 package com.learning.libraryManagement.controller;
 
+
+import com.learning.libraryManagement.model.Books;
+import com.learning.libraryManagement.service.BooksService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
 public class BooksController {
+    @Autowired
+    BooksService booksservice;
+
+
+
+    @PostMapping("/getAllBooks")
+    public List<Books> getBooks(@RequestBody List<Books> books){
+        return booksservice.getBooks(books);
+    }
+
```

Comment:
Write 2 end points one for get and one for post 

Post endpoint to save mutiple books

Reply:
added new changes in code

Reply:
done the change